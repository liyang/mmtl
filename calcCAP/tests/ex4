0.002
0.00133333333333
439500.0
439500.000001
-0.001
-0.0005
1024
60
6
8
10
5
0.0
1.0e-5
2
5
10000.0
2 3 0 0 0 0 1
DIELE 19 dieleTags1 1 -0.00095 0 0.0019 0.0002 4.3 DIELE 20 dieleTags2 2 -0.00095 0.0002 0.0019 0.0001 3.2
REC 21 recTags1 1 -0.000675 0.0002 0.00035 0.00007 REC 22 recTags2 2 -0.000175 0.0003 0.00035 0.00007 REC 23 recTags3 3 0.000325 0.0003 0.00035 0.00007




GND 24 gndTags1 1 -0.00095 -0.0001 0.0019 0.0001
