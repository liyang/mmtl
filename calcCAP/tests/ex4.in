1
3
0
0
2
-0.00095
-0.0001
0.0019
0.0001
-0.000675
0.0002
0.00035
0.00007
-0.000175
0.0003
0.00035
0.00007
0.000325
0.0003
0.00035
0.00007
-0.00095
0
0.0019
0.0002
4.3
-0.00095
0.0002
0.0019
0.0001
3.2
1024
60
6
8
10
5
0.0
1.0e-5
2
5
10000.0
