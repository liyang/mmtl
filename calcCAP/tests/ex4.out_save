142.158     -21.943     -0.928771     
-21.9431     93.6633     -18.2273     
-0.928846     -18.2273     87.9803     
