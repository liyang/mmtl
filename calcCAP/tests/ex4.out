---------------------------------------
Capacitance Values
Input file : ex4.in
---------------------------------------
142.158     -21.9432     -0.928642     
-21.943     93.6633     -18.2273     
-0.928637     -18.2273     87.9803     
Estimated Maximum Computational Error is: -0

User CPU time: 55
 Sys CPU time: 0
